packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -140,6 +140,17 @@ describe( 'Basic rendering', () => {
 		expect( searchInput ).toBeInTheDocument();
 	} );
 
+	it( 'should have aria-owns attribute to follow the ARIA 1.0 pattern', () => {
+		render( <LinkControl /> );
+
+		// Search Input UI.
+		const searchInput = screen.getByRole( 'combobox', { name: 'URL' } );
+
+		expect( searchInput ).toBeInTheDocument();
+		expect( searchInput ).not.toHaveAttribute( 'aria-controls' );
```

Comment:
I don't see a good reason to have this assertion in place. I think we can remove it. 

```suggestion
```

Reply:
I understand. thanks for clarifying. I guess it's worth adding some context as a comment to the test then 👍

Reply:
My bad, I see now you mentioned the assertion for `aria-controls`, not the one for `aria-selected`. Will add comments to clarify both.